src/main/bash/shifu

```diff
@@ -108,6 +108,17 @@ fi
 
 SHIFU_OPTS=${JAVA_OPTS:=${SHIFU_OPTS}}
 
+if [ "${SPARK_HOME}" != "" ]; then
+   if [ "${SPARK_CONF_DIR}" != "" ]; then
+        SPARK_SUBMIT=`command -v "spark-submit"`
+        if [ "${SPAK_SUBMIT}" == "" ]; then
+            SPARK_SUBMIT="${SPARK_HOME}/bin/spark-submit"
+            SPARK_EVAL_JAR=`find ${SHIFU_HOME}/lib -name "shifu-spark-eval*.jar"`
+            SPARK_CLASSPATH=`find ${SHIFU_HOME}/lib -name "*.jar" | grep -v javadoc | xargs | sed 's/ /,/g'`
+        fi
+   fi
+fi
+
```

Comment:
Add comment here to let users to set SPARK_HOME if not exist or if user would like to specify his own SPARK_HOME

Can we make all such config in the bash comment